docs/sdk/paying-for-lit/payment-delegation-db.md

```diff
@@ -0,0 +1,269 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Payment Delegation Database
+
+:::info
+Currently the Payment Delegation Database is only supported on the `habanero` and `manzano` Lit networks. Payment for usage is **not** currently required on the `cayenne` network.
```

Comment:
Cayenne will never support payment. So we can remove this info note.

Reply:
This not is more so just to be explicit for new users who don't understand that payment isn't required on `cayenne`, I'll change to: `Payment for usage is **not** required on the cayenne network.`